### PR TITLE
LVGL cookbook

### PR DESCRIPTION
**Pull request by nagyrobi**
## Description:

Cookbook originally contained in #3678, which should be merged first.

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#<esphome PR number goes here>

## Checklist:

  - [x] I am merging into `next` because this is new documentation that has a matching pull-request in [esphome](https://github.com/esphome/esphome) as linked above.  
    or
  - [ ] I am merging into `current` because this is a fix, change and/or adjustment in the current documentation and is not for a new component or feature.

  - [x] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

**Comment by nagyrobi** [comment]
Due to too many conflicts, the doc has been resumbmitted as a new PR: https://github.com/esphome/esphome-docs/pull/4115

File: cookbook/lvgl.rst [hunk 1]
```diff
@@ -0,0 +1,2242 @@
+.. _lvgl-cookbook:
+
+LVGL: Tips and Tricks
+=====================
+
+.. seo::
+    :description: Recipes for common use cases of LVGL Displays with ESPHome
+    :image: /images/lvgl.png
+
+Here are a couple recipes for various interesting things you can do with :ref:`lvgl-main` in ESPHome.
```
Comment (kbx81) [inline cookbook/lvgl.rst]:
```suggestion
Here are a couple recipes for various interesting things you can do with :doc:`/components/lvgl/index` in ESPHome.
```

File: cookbook/lvgl.rst [hunk 2]
```diff
@@ -0,0 +1,2242 @@
...
+                then:
+                  - lvgl.resume:
+                  - lvgl.widget.redraw:
+
+You can combine it with the previous example to turn off the backlight, so the users don't actually notice this.
+
+See Also
+--------
+
+- :ref:`lvgl-main`
```
Comment (kbx81) [inline cookbook/lvgl.rst]:
```suggestion
- :doc:`/components/lvgl/index`
```

**Review by kbx81 — CHANGES_REQUESTED** [review]
Fix a couple links

**Review by jesserockz — DISMISSED** [review]

**Review by jesserockz — APPROVED** [review]

**Pull request merged**